pkg/kubelet/prober/worker.go

```diff
@@ -219,7 +221,14 @@ func (w *worker) doProbe(ctx context.Context) (keepGoing bool) {
 		return false
 	}
 
-	c, ok := podutil.GetContainerStatus(status.ContainerStatuses, w.container.Name)
+	containerStatuses := status.ContainerStatuses
```

Comment:
Does this allocate where it wouldn't have before?  Or do we have the same rough numbers of allocations (probe has been previously optimized to avoid allocations, which is why i am asking)?

Reply:
fixed to make sure not to allocate where it wouldn't have before.